src/analytics.ts

```diff
@@ -74,6 +74,9 @@ export const analyticsMiddleware = async (args: {
       ci: isCi(),
       githubEnvVars: getGithubEnvVars(process.env),
     },
+    context: {
+      direct: true,
```

Comment:
does that mean that segment will collect addresses itself based on network request? I wonder which one is better in terms of accuracy...